frontend/src/components/status/SystemCheck.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import { Glyphicon, Label } from 'react-bootstrap';
+
+const styles = {
+  container: {
+    fontSize: 18,
+    paddingLeft: 30,
+  },
+  passing: {
+    color: 'green',
+    padding: '10px 40px 0px 0px',
+  },
+  failing: {
+    color: 'red',
+    padding: '10px 40px 0px 0px',
+  },
+  glyphicon: {
+    marginRight: 10,
+  },
+  icon: {
+    float: 'right',
+    fontSize: 24,
+    paddingLeft: 15,
+  },
+  description: {
+    textAlign: 'left',
+    paddingTop: 25,
+  },
+  currentSpecs: {
+    fontSize: 14,
+    paddingTop: 8,
+  },
+};
+
+// This component is intended for internal use only and does not use styles or
+// components of public facing CAT system.
+class SystemCheck extends Component {
+  static propTypes = {
+    description: PropTypes.string.isRequired,
+    isPassing: PropTypes.bool.isRequired,
+    screenResolutionDetected: PropTypes.string,
```

Comment:
instead of 3 variable names, I think it would make more sense to have a prop
```
settingDetails: PropTypes.string,
```

and then on line 60 and 69, you would just have `{settingDetails}`